src/creator.coffee

```diff
@@ -297,3 +306,26 @@ SurveyWidget.factory 'Resource', ['$sanitize', ($sanitize) ->
 		questions: [{ text: question }]
 		answers: item.answers
 ]
+
+SurveyWidget.service 'sanitizeHelper', [() ->
+	SANITIZE_CHARACTERS =
+		'>' : '&gt;',
+		'<' : '&lt;',
+		'&' : '&amp;',
+		'"' : '&#34;'
+
+	sanitize = (input) ->
+		unless input then return
+		for k, v of SANITIZE_CHARACTERS
+			input = input.replace k, v
```

Comment:
str.replace only finds and replaces the first instance of a string in a string - for example, if a string is is '&&' this will result in '&amp;&'.

`.replace(/&/g, '&amp;')` would grab them all.